### PR TITLE
Move voting from snapshot

### LINKED ISSUE #398: Move `helpers/voting` from snapshot into snapshot.js

### PR DESCRIPTION
**Pull request by aiinkiestism**
Fixes #398  .

Changes proposed in this pull request:
- Move helpers/voting from snapshot into snapshot.js

I also moved getNumberWithOrdinal from snapshot into snapshot.js because it's only imported from voting/rankedChoice.ts

See [this PR](https://github.com/snapshot-labs/snapshot/pull/984) too

### CODE REVIEW

**Review by samuveth — APPROVED** [review]
Looks good!

**Comment by samuveth** [comment]
Can you resolve conflicts @aiinkiestism

File: src/utils.ts [hunk 1]
```diff
@@ -158,6 +164,7 @@ export default {
   sendTransaction,
   getScores,
   validateSchema,
+  getNumberWithOrdinal,
   getProvider,
```
Comment (bonustrack) [inline src/utils.ts:168]: Can you export voting classes here too?
```suggestion
  getNumberWithOrdinal,
  voting,
  getProvider,
```

**Comment by samuveth** [comment]
@aiinkiestism can you fix the build error?

**Comment by aiinkiestism** [comment]
@samuveth 
Didn't notice, thanks!
That was a typescript error.

**Review by bonustrack — APPROVED** [review]

**Comment by bonustrack** [comment]
Looks good thanks @aiinkiestism !

**Pull request merged**